Semver can't be resolved because Semver pom.xml is missing from JFrog

If your root build is just this:
```
plugins {
    id("org.jlleitschuh.gradle.ktlint-idea") version "6.2.1"
}
```

You get the following exception:
```
./gradlew tasks

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'root-dependency-manager'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find net.swiftzer.semver:semver:1.1.0.
     Searched in the following locations:
       - file:/Users/jonathanleitschuh/.gradle/caches/5.0-rc-1/embedded-kotlin-repo-1.3.0-2/repo/net/swiftzer/semver/semver/1.1.0/semver-1.1.0.jar
       - https://plugins.gradle.org/m2/net/swiftzer/semver/semver/1.1.0/semver-1.1.0.pom
     Required by:
         project : > org.jlleitschuh.gradle.ktlint-idea:org.jlleitschuh.gradle.ktlint-idea.gradle.plugin:6.2.1 > gradle.plugin.org.jlleitschuh.gradle:ktlint-gradle:6.2.1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

This is because the semver plugin isn't published to the Gradle plugin portal.
We should consider embedding that project's one source file in our build as a vendored dependency.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,15 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
-### Added
-  - ?
-### Changed
-  - ?
-### Removed
-  - ?
+## [6.3.1] - 2018-11-27
 ### Fixed
-  - ?
+  - Updated SemVer dependency to `1.1.1` version (#162)
 
 ## [6.3.0] - 2018-11-06
 ### Added

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 **Provides a convenient wrapper plugin over the [ktlint](https://github.com/shyiko/ktlint) project.**
 
-Latest plugin version: [6.3.0](/CHANGELOG.md#630---2018-11-06)
+Latest plugin version: [6.3.1](/CHANGELOG.md#631---2018-11-27)
 
 [![Join the chat at https://gitter.im/ktlint-gradle/Lobby](https://badges.gitter.im/ktlint-gradle/Lobby.svg)](https://gitter.im/ktlint-gradle/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/JLLeitschuh/ktlint-gradle.svg?branch=master)](https://travis-ci.org/JLLeitschuh/ktlint-gradle)

--- a/plugin/build.gradle.kts
+++ b/plugin/build.gradle.kts
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "org.jlleitschuh.gradle"
-version = "6.4.0-SNAPSHOT"
+version = "6.3.1"
 
 repositories {
     jcenter()

